modules/alloy/src/test/ts/browser/position/SelectionInDocPositionTest.ts

```diff
@@ -10,6 +11,26 @@ import * as ChainUtils from 'ephox/alloy/test/ChainUtils';
 import * as PositionTestUtils from 'ephox/alloy/test/PositionTestUtils';
 import * as Sinks from 'ephox/alloy/test/Sinks';
 
+const toDomRange = (cursorRange: Cursors.CursorRange): Range => {
```

Comment:
All functions should be within the `UnitTest.asynctest` block, or should be in a separate utility file.

- [ ] Move to separate file or to be beside `cSetupAnchor`